.claude/hooks/metrics_summary.sh

```
@@ -0,0 +1,135 @@
+      read_lines_saved: 0,
+      diff_count: 0,
+      warnings_count: 0,
+      sessions: {}
+    };
+    
+    # Process item
+    if $item.tool == "Read" then
+      .read_count += 1 |
+      .read_lines_saved += ($item.original_lines - $item.modified_lines) |
+      if $item.session_id then
+        # Initialize session object if it doesn't exist
```

Shellcheck (0.11.0)

[warning] This apostrophe terminated the single quoted string!

(SC1011)